src/main/java/org/folio/circulation/domain/UpdateLoan.java

```diff
@@ -46,8 +46,11 @@ public UpdateLoan(Clients clients,
     Loan loan = request.getLoan();
 
     if (request.getRequestType() == RequestType.RECALL && loan != null) {
+      final var finalRequestAndRelatedRecords
+        = requestAndRelatedRecords.withRecalledLoanPreviousDueDate(loan.getDueDate());
```

Comment:
![image](https://user-images.githubusercontent.com/13659775/162573898-126c4468-c50d-4242-8f5d-b477d61e234c.png)
I got this suggestion, so that used another variable